recipes/glue-qt/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "glue-qt" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/glue-qt-{{ version }}.tar.gz
+  sha256: 26a7f10e8bb58426512921355a0b9078698170fcc66504b2174969e1d1073d42
+
+build:
+  skip: true  # [py<38]
```

I'm guessing there are no compiled parts, right? If so we can make this noarch.

```suggestion
  noarch: python
```